docker/entrypoint.sh

```diff
@@ -79,6 +79,11 @@ then
     ETCD_CLUSTER="127.0.0.1:4001"
 fi
 
+mkdir -p ~postgres/.config/patroni
+cat > ~postgres/.config/patroni/patronictl.yaml <<__EOF__
+{dcs_api: 'etcd://${ETCD_CLUSTER}', namespace: /service/}
+__EOF__
+
```

Comment:
It's valid yaml, it was generated using yaml.dump:

    >>> print( yaml.dump({'api_url':'etcd://localhost:4001', 'namespace': '/service/'}) )
    {api_url: 'etcd://localhost:4001', namespace: /service/}

  I chose yaml to have some similarity with the stups tooling, which all save their config in `~/.config/<tool>/tool.yaml` as well.